src/uu/ls/src/ls.rs

```diff
@@ -255,6 +255,18 @@ fn sort_entries(entries: &mut Vec<PathBuf>, options: &getopts::Matches) {
     }
 }
 
+#[cfg(windows)]
+fn is_hidden(file_path: &std::path::PathBuf) -> std::io::Result<bool> {
+    let metadata = fs::metadata(file_path)?;
+    let attr = metadata.file_attributes();
+
+    if (attr & 0x2) > 0 {
+        Ok(true)
+    } else {
+        Ok(false)
+    }
```

Comment:
Did not try if it works but what about:
```suggestion
    Ok(attr & 0x2) > 0)
```

Reply:
I can check sure, but what comments are you referring to?